catalog/transformer/catalog-transformer-zip/src/main/java/org/codice/ddf/catalog/transformer/zip/ZipCompression.java

```diff
@@ -247,19 +249,24 @@ private void writeDataToZip(ZipOutputStream zipOutputStream, InputStream inputSt
   private BinaryContent getBinaryContentFromZip(String filePath)
       throws CatalogTransformerException {
     BinaryContent binaryContent;
+    InputStream fileInputStream;
+    File zipFile = new File(filePath);
     try {
-      File zipFile = new File(filePath);
-      InputStream fileInputStream = new ZipInputStream(new FileInputStream(zipFile));
+      fileInputStream = new ZipInputStream(new FileInputStream(zipFile));
       binaryContent = new BinaryContentImpl(fileInputStream);
-      jarSigner.signJar(
-          zipFile,
-          System.getProperty(SystemBaseUrl.EXTERNAL_HOST),
-          System.getProperty("javax.net.ssl.keyStorePassword"),
-          System.getProperty("javax.net.ssl.keyStore"),
-          System.getProperty("javax.net.ssl.keyStorePassword"));
     } catch (FileNotFoundException e) {
       throw new CatalogTransformerException("Unable to get ZIP file from ZipInputStream.", e);
     }
+
+    jarSigner.signJar(
+        zipFile,
+        System.getProperty(SystemBaseUrl.EXTERNAL_HOST),
```

Comment:
❗️ Missed this one.

Reply:
I only added ones that were by default denied in the default.policy. Should I include this one?

Reply:
It is always best to add it.

Reply:
Okay I'll make the change now